proposals/build-cli-options.md

```diff
@@ -0,0 +1,50 @@
+# [Proposal] Allow additional cli options to pass to the build command
+
+Related to:
+- https://github.com/devcontainers/spec/discussions/301
+- https://github.com/devcontainers/spec/pull/324
+- https://github.com/microsoft/vscode-remote-release/issues/3545
+
+## Goal
+
+Extend the build section of the configuration with an option to pass additional arguments via cli.
+
+## Proposal
+
+Introduce the property `cliOptions` to the `buildOptions` in the schema as type `Array (string)`. This field contains elements that are directly passed as cli-arguments to the `build` command. It is similar to `runArgs` property which contains cli-arguments passed to the `run` command.
+
+###  Execution
+
+When a dev container is built, all entries from the `cliOptions` are passed to the `build` command at the end.
```

Comment:
Change "at the end" to "after all other options" since the context path will still go last?

Reply:
Good catch. True of course, adjusted it.